### ISSUE TITLE
controlled input in cell and header - lose focus

### ISSUE THREAD

**Issue by ecadagiani** [react-bootstrap-table/react-bootstrap-table2#1387]
### TL;DR:
I wan't to create table with controlled input in cell and header. All row editable at the same time, so without editableRow. But if i use formatter, the cell re-mount at each modification, and input loose focus (I wan't to avoid dirty tricks).

### TL:
Hi everyone, first of all, and like the other issue, thanks for this nice component. I would like to congratulate simplicity of use, and the possible customization of this API.

I migrate all the table of my app from reactTable v6 to react-bootstrap-table2. For almost all the tables everything went smoothly. But i have some "hard head" table.

I used reactTable in form, to create a element list with pagination, with controlled input in cell and header.
Exemple:
![Capture d’écran 2020-06-04 à 13 57 10](https://user-images.githubusercontent.com/1338218/83753909-6b4ad800-a66b-11ea-8620-7344a3b87ab7.png)
![Capture d’écran 2020-06-04 à 13 57 33](https://user-images.githubusercontent.com/1338218/83753934-7271e600-a66b-11ea-9d6a-abdbf6bb7535.png)

With reactTable i used some dirty tricks to avoid re-mount. But with react-bootstrap-table2, I did not succeed at all (All row editable at the same time, without editableRow. With input in formatter, the cell re-mount at each modification, and input loose focus).
As it is a refactor, i wan't to avoid dirty tricks.

### My question is:
Can i use react-bootstrap-table2 to replace this table (with clean modification) ?
If it is not possible, can i still use the PaginationProvider of react-bootstrap-table2 (to have a continuity with other table) ?
